app/Policies/CommentPolicy.php

```diff
@@ -47,19 +47,6 @@ public function viewAny(?User $user, Model $commentable): bool
     public function create(?User $user, ?Model $commentable = null, ?int $articleType = null): bool
     {
         if ($user?->isMuted()) {
-            // Even when muted, developers may still comment on tickets for their own achievements.
-            // TODO this is silly. delete all of this.
-            if ($commentable !== null && $commentable instanceof \App\Models\Ticket) {
-                $commentable->loadMissing(['achievement.developer']);
-
-                $didAuthorAchievement = $commentable->achievement->developer->id === $user->id;
-
-                return
-                    $didAuthorAchievement
-                    && $commentable->is_open
-                    && $user->hasAnyRole([Role::DEVELOPER_STAFF, Role::DEVELOPER]);
-            }
```

Comment:
This was removed in another PR where I suggested it was still needed. I realize there will be conflicts, but you should only remove the DEVELOPER_STAFF check at this time.

Reply:
Per the discussion mentioned here: https://github.com/RetroAchievements/RAWeb/pull/2967#discussion_r1899637199 I'm thinking maybe we should keep this code all deleted in this branch.